.circleci/config.yml

```diff
@@ -1,5 +1,6 @@
 version: 2.1
-
+orbs:
+  ship: auth0/ship@dev:b224b98
```

Comment:
Will update this later, currently using a development build until the java bits are final and released